practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/client/RoleServiceClient.java

```diff
@@ -1,17 +1,20 @@
 package uk.ac.ox.ndph.mts.practitioner_service.client;
 
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.web.reactive.function.client.WebClient;
 
 @Component
 public class RoleServiceClient extends AbstractEntityServiceClient {
 
-    public RoleServiceClient(final WebClient.Builder webClientBuilder,
-                             @Value("${role.service.uri}") String serviceUrlBase) {
+    @Autowired
+    public RoleServiceClient(WebClient.Builder webClientBuilder) {
+        this(webClientBuilder, "http://role-service");
+    }
```

Comment:
I think it is better to have the url/service name in discovery, as part of the configuration

Reply:
I reverted it to use config file, @eladiw, @tamirkamara  WDYT?